### PR TITLE
design section-works cards and section-about-me 

### PR DESCRIPTION
**Pull request by jubaer919**
i completed the section work. i design the cards exactly like figma. gives the button hover effect. also completed about me section exactly like the given design.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -36,6 +36,136 @@ <h1 class="heading-primary">I’m Jubaer. <span class="heading-primary__sapan">G
...
+                <h2 class="heading-primary">About Myself</h2>
+                <p class="section-hero__paragraph">Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+                <a href="#" class="section-hero__link">LET’S CONNECT</a>
+                <div class="section-hero__social-media">
+                    <img src="./images/twiter.svg" alt="twiter logo" class="twiter-logo">
+                    <img src="./images/in.svg" alt="linked-in logo">
+                    <img src="./images/m.svg" alt="m logo">
+                    <img src="./images/victory.svg" alt="victory logo">
+                    <img src="./images/github.svg" alt="github logo">
+                </div>
```
Comment (mckent05) [inline index.html]: - To help improve the semantics of your site, I suggest wrapping this section in a `ul` tag and each of the images in a corresponding `li` tag. Please fix this 🙏

File: index.html [hunk 2]
```diff
@@ -36,6 +36,136 @@ <h1 class="heading-primary">I’m Jubaer. <span class="heading-primary__sapan">G
...
+                        </div>
+                        <hr>
+                        <div class="language-des">
+                            <h3 class="language-title">Skills</h3>
+                            <img src="./images/arrow-right-svgrepo-com.svg" alt="logo" class="arrow-logo">
+                        </div>
+                        <hr>
+                    </div>
+                </div>
+            </div>
```
Comment (mckent05) [inline index.html]: - To help improve the semantics of your site, I suggest wrapping this section in a `ul` tag and wrap each corresponding text i.e `Languages, Frameworks, Skills` in a corresponding `li` tag. Also, wrap the technologies in a `ul` tag and wrap each of the technology in a corresponding `li` tag. Please fix this 🙏 . You might find the text below useful
```
<ul>
  <li>
	<div>
	    <p>Languages</p>
	    <img>
	</div>
	<ul>
	    <li>
		<img>
		<p>Javascript</p>
	    <li>
	    <li>
		<img>
		<p>HTML</p>
	    <li>
            <li>
		<img>
		<p>CSS</p>
	    <li>
	</ul>
  </li>
  <li>
     <p>FrameWorks</p>
     <img>
  </li>	
  <li>
     <p>Skills</p>
     <img>
  </li>
</ul>```

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @jubaer919, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 
- Your project looks good, good job 👏 👏 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Yerimah — APPROVED** [review]
# STATUS: APPROVED 🔥 

Hi @jubaer919, 👋  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Celebrate We The Best GIF by Booksmart

## Highlights
- No linter error ✅ 
- GitHub flow ✅ 
- Descriptive PR summary and title

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**